docs/database-engine/availability-groups/windows/distributed-availability-groups.md

```diff
@@ -57,6 +57,8 @@ Because the distributed availability groups feature did not exist in SQL Server
 > [!NOTE]
 > Distributed availability groups can not be configured with Standard edition or mix of Standard and Enterprise edition.
 
+---this needs to be changed to reflect that standard edition is now supported start in SQL 2019 CU17 
```

Comment:
```suggestion
```
Moving the contributor's suggestion to a comment:

"---this needs to be changed to reflect that standard edition is now supported start in SQL 2019 CU17 "